MobileFoodSchedules/MobileFoodSchedules/Models/MobileFoodPermit.cs

```diff
@@ -1,10 +1,4 @@
-﻿// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
-//    var mobileFoodPermit = MobileFoodPermit.FromJson(jsonString);
```

Comment:
Good idea to remove commented code.  This is considered clutter in modern version control systems.  Plus, you can always go back in history to see what used to be there.